services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy

```diff
@@ -63,4 +64,35 @@ class BandwidthSpec extends BaseSpecification {
         cleanup: "Delete created flow"
         flow?.id && northboundService.deleteFlow(flow.id)
     }
+
+    def "Able to exceed bandwidth limit on ISL when creating a flow"() {
+        given: "Two active switches"
+        def switches = topology.getActiveSwitches()
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
+                .find { Switch src, Switch dst -> src != dst }
+        assert srcSwitch && dstSwitch
+
+        when: "Create a flow with a bandwidth that exceeds available bandwidth on ISL (ignore_bandwidth = false)"
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        def maxAvailableBw = (northboundService.getAllLinks()*.availableBandwidth).max()
```

Comment:
This is not 100% accurate. You pick max bandwidth of the whole system, while in fact you should select maxBandwidth across all available paths